stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -26,7 +26,7 @@
     private static final int END_INDEX_AMEX = 11;
 
     private CardNumberEditText mCardNumberEditText;
-    private EditText mCvcNumberEditText;
+    private DeleteWatchEditText mCvcNumberEditText;
```

Comment:
I actually am going to do exactly that in the next diff (or maybe the one after). For a while I was trying to separate the different functionalities into different classes for more granular reuse, but it was going to cause a massive tower of inheritance.